src/pieces/commandManager.ts

```diff
@@ -85,6 +85,10 @@ async function runCommand(msg: Message) {
 
 	const command = getCommand(msg.client, commandName);
 	if (!command || command.enabled === false) return;
+	if (msg.channel.id !== CHANNELS.SAGE && command.restricted) {
+		msg.author.send('This command is only available in #sages_place.');
+		return msg.delete();
```

Comment:
as per Josh's comment, this wouldn't be needed, or even functional, using the reply function once slash commands are in. We can just send the response to the channel as a temporary measure.